admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -154,6 +154,121 @@ In openSUSE, modify the ``/usr/sbin/start_apache2`` file::
 
 Restart Apache, open your ownCloud Admin page and start creating SMB/CIFS mounts.
 
+
+Create Shares programmatically
+------------------------------
+
+Some users want to create new shares by command line scripts. There are two different approaches.
```

Comment:
``Some users want`` --> ``Some admins want``
Users do not have command line access to the instance...